From e940e26d43cc463f194a30d04aa18811ad4f9391 Mon Sep 17 00:00:00 2001
From: Barak Avrahami <19807724+andecy64@users.noreply.github.com>
Date: Sun, 3 May 2020 13:07:04 +0300
Subject: [PATCH] Changed firefox tag

---
 config.def.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.def.h b/config.def.h
index 1c0b587..75851f0 100644
--- a/config.def.h
+++ b/config.def.h
@@ -27,8 +27,8 @@ static const Rule rules[] = {
 	 *	WM_NAME(STRING) = title
 	 */
 	/* class      instance    title       tags mask     isfloating   monitor */
-	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
+	// { "Gimp",     NULL,       NULL,       0,            1,           -1 },
+	{ "Firefox",  NULL,       NULL,       2 << 9,       0,           -1 },
 };
 
 /* layout(s) */
-- 
2.26.2

